S12-enums/basic.t

```diff
@@ -252,4 +252,11 @@ is-deeply do { my enum Foos (a => <42>); a.Str }, 'a',
     'Pair elements in the list given to enum declaration work';
 }
 
+{ # RT #129142
+    enum FF <zero one two three>;
+    enum GG <fee fie foo fum>;
+    lives-ok { FF(GG(2)).perl }, 'Coercing an enum from a coercion of an enum from an int works';
```

Comment:
Hey @zoffixznet, what do you think, does it even make sense to have `lives-ok` there when testing for compile-time errors?

Reply:
Yes, if you stick the entire code in question into a string and use [`eval-lives-ok`](https://docs.perl6.org/language/testing#index-entry-eval-lives-ok-eval-lives-ok%28%24string%2C_%24description%3F%29). Alternatively, you can just use [`pass`](https://docs.perl6.org/language/testing#index-entry-pass-flunk-pass%28%24description%3F%29%2C_flunk%28%24description%3F%29) after the code, since it'd crash before reaching the `pass` if the bug were still there.